_plugins/markdown.rb

```diff
@@ -0,0 +1,23 @@
+=begin
```

Comment:
Outta curiosity, did you yank this from somewhere or does dat bones be writing Ruby?